howso/return_typing.amlg

```diff
@@ -496,130 +496,194 @@
 						}
-						"smape" {
+						description "The symmetric mean absolute percentage error with added the min gap / 2 to the actual and predicted values."
+					}
+					"smape" {
+						type "assoc"
+						additional_indices {
 							type ["number" "null"]
-							description "The symmetric mean absolute percentage error of predicting the feature."
 						}
-						"mae" {
+						description "The symmetric mean absolute percentage error of predicting the feature."
+					}
+					"mae" {
+						type "assoc"
+						additional_indices {
 							type ["number" "null"]
-							description "The mean absolute error of predicting the feature."
 						}
-						"recall" {
+						description "The mean absolute error of predicting the feature."
+					}
+					"recall" {
+						type "assoc"
+						additional_indices {
 							type ["number" "null"]
-							description "The recall of predicting the feature."
 						}
-						"precision" {
+						description "The recall of predicting the feature."
+					}
+					"precision" {
+						type "assoc"
+						additional_indices {
 							type ["number" "null"]
-							description "The precision of predicting the feature."
 						}
-						"accuracy" {
+						description "The precision of predicting the feature."
+					}
+					"accuracy" {
+						type "assoc"
+						additional_indices {
 							type ["number" "null"]
-							description "The accuracy of predicting the feature."
 						}
-						"r2" {
+						description "The accuracy of predicting the feature."
+					}
+					"r2" {
+						type "assoc"
+						additional_indices {
 							type ["number" "null"]
-							description "The R^2 of predicting the feature."
 						}
-						"rmse" {
+						description "The R^2 of predicting the feature."
+					}
+					"rmse" {
+						type "assoc"
+						additional_indices {
 							type ["number" "null"]
-							description "The RMSE of predicting the feature."
 						}
-						"spearman_coeff" {
+						description "The RMSE of predicting the feature."
+					}
+					"spearman_coeff" {
+						type "assoc"
+						additional_indices {
 							type ["number" "null"]
-							description "The Spearman's coefficient of predicting the feature."
 						}
-						"mcc" {
+						description "The Spearman's coefficient of predicting the feature."
+					}
+					"mcc" {
+						type "assoc"
+						additional_indices {
 							type ["number" "null"]
-							description "The MCC of predicting the feature."
 						}
-						"missing_value_accuracy" {
+						description "The MCC of predicting the feature."
+					}
+					"missing_value_accuracy" {
+						type "assoc"
+						additional_indices {
 							type ["number" "null"]
-							description "The proportion of missing values that were correctly predicted as missing for the feature."
 						}
-						"confusion_matrix" {ref "ConfusionMatrix"}
+						description "The proportion of missing values that were correctly predicted as missing for the feature."
+					}
+					"confusion_matrix" {
```

Comment:
Missing a description

Reply:
Added a short description as a more complex one is already contained in the definition of the ConfusionMatrix type.